pdf_to_ppt/requirements.txt

```diff
@@ -1,5 +1,10 @@
+altgraph==0.17.2
+fbs==1.1.2
+future==0.18.2
 lxml==4.8.0
+macholib==1.16
 pdf2image==1.16.0
+pefile==2021.9.3
 Pillow==9.0.1
 python-pptx==0.6.21
-XlsxWriter==3.0.2
+XlsxWriter==3.0.3
```

Comment:
check out pip-tools it can keep track of your library’s

Reply:
i'll add that in a future PR